app/app.go

```diff
@@ -582,16 +610,19 @@ func NewShentuApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 			tmos.Exit(err.Error())
 		}
 	}
```

Comment:
Did we double check there isn't any upgrade handler needed here?

Reply:
Also let's set the default checking period to 0 so it doesn't check the invariants by default.

Reply:
Its default is already set to 0 in cosmos-sdk/server/start.go
```
168 >-------cmd.Flags().Uint(FlagInvCheckPeriod, 0, "Assert registered invariants every N blocks")
```
Did I find the right place?

Reply:
Yes then I think it's good.